README.md

```diff
@@ -8,3 +8,19 @@ Add all of your Research, Meeting Agendas and everything project related.
 ## Contributing
 
 Please see the [contribution guidelines](CONTRIBUTING.md).
+
+
+## What is this repositry used for?
+
+This documentation repo is used to collate all information in regards to the Thoth-Tech Project. 
```

Comment:
```suggestion
This documentation repo is used to collate all information regarding Thoth Tech Products 
```

Reply:
Hi Talia, No worries at all. Thank you for your comments